server/lib/fusor/password_filter.rb

```diff
@@ -0,0 +1,85 @@
+##
+# PasswordFilter
+# ================
+# Extracts passwords from deployment objects.
+# Provides functionality for filtering sensitive data (passwords) from text.
+
+class PasswordFilter
+  def self.password_cache
```

Comment:
Any reason we made this static and store state? Have we tried multiple users logged into the satellite trying to create deployments? Will it use the same instance?